shopinvader/tests/test_backend.py

```diff
@@ -137,15 +137,20 @@ def test_add_new_lang_rebind(self):
         """
         self._bind_all_product()
         self._bind_all_category()
-        binded_variants_count = self.env["shopinvader.variant"].search_count([])
-        binded_categories_count = self.env["shopinvader.category"].search_count([])
+        domain = [("backend_id", "=", self.backend.id)]
+        binded_variants_count = (
+            self.env["shopinvader.variant"]
+            .with_context(bind_products_immediately=True)
```

Comment:
is this ctx flag really needed?

Reply:
I just removed it. It was a test about a binding issue.